src/async-input.js

```diff
@@ -77,4 +77,5 @@ export const AsyncInputClasses = {
    * and use async-input elements.
    */
   'ASYNC_INPUT': 'i-amphtml-async-input',
+  'ASYNC_REQUIRED_ACTION': 'i-async-require-action',
```

Comment:
This attribute is added by the page author, right? If so, we should remove the `i-*` prefix which means "internal".

Also, is it valid AMP?

Reply:
the tag would be added by the extension itself; if it determines that required actions exist, it will add the tag within the class list of the payment extension

Reply:
Ok, let's make it `i-amphtml-async-require-action` then.